code/index.html

```diff
@@ -1,30 +1,227 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <title>Dreamzzz</title>
+    <link type="text/css" rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;600&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <!--VIDEO-->
+
+    <video loop muted autoplay preload="auto">
+        <source src="./videos/NYcleaningconfetti.mp4" video type="video/mp4" />
+    </video>
+
+    <!--HEROIMAGE, HEROTEXT AND OVERLAY TEXT-->
+
+    <div class="hero-image">
+        <div class="hero-text">
+            <h1>You have the dream.<br>We make it happen.</h1>
+        </div>
+    </div>
+
+    <div class="overlay-text">
+        <h1>You have the dream.<br>We make it happen.</h1>
```

Comment:
Instead of using the `<br>`-tag please try to either style it with CSS or divide it in to two separate h1